### PR TITLE
[Common References] Add references

### PR DESCRIPTION
**Pull request by pgoyal01**
Partially Fixes Issue #2389 

Added external references -- please suggest any missing. Thanks

### CODE REVIEW

**Comment by walterkozlowski** [comment]
@pgoyal01, have you added references from PR#2427?

File: doc/common/references.md [hunk 1]
```diff
@@ -37,13 +38,64 @@
...
+
+<a name="1.5"></a>
+## Security Related References
+
+|Ref	     | Doc Number	            |Title          |
+|----------|------------------------|---------------|
+| | | CSA's Information Security Management through Reflexive Security. Available at https://cloudsecurityalliance.org/artifacts/information-security-management-through-reflexive-security/. |
+| | | ETSI Industry Specification Group Network Functions Virtualisation (ISG NFV).  Available at https://www.etsi.org/committee/1427-nfv. |
+| | | ETSI Cyber Security Technical Committee (TC CYBER).  Available at https://www.etsi.org/committee/cyber
+| | ETSI GS NFV-SEC 014 V3.1.1 (2018-04) |  Network Functions Virtualisation (NFV) Release 3; "NFV Security; Security Specification for MANO Components and Reference points";  ETSI GS NFV-SEC 014 V3.1.1 (2018-04). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/014/03.01.01_60/gs_NFV-SEC014v030101p.pdf. |
```
Comment (ulikleber) [inline doc/common/references.md:79]:
```suggestion
| | ETSI GS NFV-SEC 014 V3.1.1 |  Network Functions Virtualisation (NFV) Release 3; "NFV Security; Security Specification for MANO Components and Reference points";  ETSI GS NFV-SEC 014 V3.1.1 (2018-04). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/014/03.01.01_60/gs_NFV-SEC014v030101p.pdf. |
```
Suggest to omit the date in the Doc Number column.

File: doc/common/references.md [hunk 2]
```diff
@@ -37,13 +38,64 @@
...
+| | ETSI GS NFV-SEC 021 V2.6.1 (2019-06) | Network Functions Virtualisation (NFV) Release 2; Security; "VNF Package Security Specification"; ETSI GS NFV-SEC 021 V2.6.1 (2019-06). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/021/02.06.01_60/gs_nfv-sec021v020601p.pdf. |
```
Comment (ulikleber) [inline doc/common/references.md:80]:
```suggestion
| | ETSI GS NFV-SEC 021 V2.6.1 | Network Functions Virtualisation (NFV) Release 2; Security; "VNF Package Security Specification"; ETSI GS NFV-SEC 021 V2.6.1 (2019-06). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/021/02.06.01_60/gs_nfv-sec021v020601p.pdf. |
```
Suggest to omit the date in the Doc Number column.

File: doc/common/references.md (hunk 1 above)
Comment (ulikleber) [inline doc/common/references.md:79]:
```suggestion
| | ETSI GS NFV-SEC 014 V3.1.1 |  Network Functions Virtualisation (NFV) Release 3; "NFV Security; Security Specification for MANO Components and Reference points";  ETSI GS NFV-SEC 014 V3.1.1 (2018-04). Available at https://www.etsi.org/deliver/etsi_gs/NFV-SEC/001_099/014/03.01.01_60/gs_NFV-SEC014v030101p.pdf. |
```
Suggest to omit the date in the Doc Number column.

File: doc/common/references.md [hunk 3]
```diff
@@ -37,13 +38,64 @@
...
+| | O-RAN.WG6.AAL-GAnP-v01.00 | "O-RAN Acceleration Abstraction Layer General Aspects an Principles 1.0", November 2020; O-RAN.WG6.AAL-GAnP-v01.00. Available at https://www.o-ran.org/specifications. |
+| | ETSI TS 123 501 V16.6.0 | "System architecture for the 5G System (5GS)". ETSI TS 123 501 V16.6.0 (2020-10) (3GPP TS 23.501 version 16.6.0 Release 16). Available at https://www.etsi.org/deliver/etsi_ts/123500_123599/123501/16.06.00_60/ts_123501v160600p.pdf. |
+
+<a name="1.5"></a>
+## Security Related References
+
+|Ref	     | Doc Number	            |Title          |
+|----------|------------------------|---------------|
+| | | CSA's Information Security Management through Reflexive Security. Available at https://cloudsecurityalliance.org/artifacts/information-security-management-through-reflexive-security/. |
+| | | ETSI Industry Specification Group Network Functions Virtualisation (ISG NFV).  Available at https://www.etsi.org/committee/1427-nfv. |
```
Comment (ulikleber) [inline doc/common/references.md:77]: This is a very common reference - even more common than all the NFV documents listed in common references ;-). Suggest to omit here.

**Review by ulikleber — APPROVED** [review]

File: doc/common/references.md (hunk 3 above)
Comment (pgoyal01) [reply doc/common/references.md:77]: @ulikleber The intent is to capture all references that are spelled out in the documents.

File: doc/common/references.md (hunk 1 above)
Comment (pgoyal01) [reply doc/common/references.md:79]: @ulikleber The references in ETSI documents includes the dates

Comment (pgoyal01) [reply doc/common/references.md:79]: @ulikleber The references in ETSI documents includes the dates

File: doc/common/references.md [hunk 4]
```diff
@@ -37,13 +38,69 @@
...
 <a name="1.2"></a>
-## RM References
+## Cloud Native and Kubernetes References
+
+|Ref	     | Doc Number	            |Title          |
+|--------  |------------------------|---------------|
+| | | "Extended Cloud Native Principles". Available at https://networking.cloud-native-principles.org/cloud-native-principles. |
+| | | "DANM". Available at https://github.com/nokia/danm. |
+| | | "Kubernetes Container Runtime Interface (CRI)". Available at https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/. |
+| | | "Multus". Available at https://github.com/intel/multus-cni. |
```
Comment (petorre) [inline doc/common/references.md:48]:
```suggestion
| | | "Multus". Available at https://github.com/k8snetworkplumbingwg/multus-cni. |
```

**Review by petorre — APPROVED** [review]

**Review by gkunz — APPROVED** [review]

File: doc/common/references.md (hunk 4 above)
Comment (pgoyal01) [reply doc/common/references.md]: @petorre Thanks - Accepted. FYI, in RA2 [3.2.2 Container Networking Services](https://github.com/cntt-n/CNTT/blob/master/doc/ref_arch/kubernetes/chapters/chapter03.md#322-container-networking-services), in the bullets before Table 3-1, the link for Multus is to the intel link.

**Pull request merged**